src/iitkgp_erp_login/erp.py

```diff
@@ -42,6 +42,10 @@ class ErpCreds(TypedDict):
     "Security Question"
 
 class ErpLoginError(Exception):
+    def __init__(self, message: str):
+        super().__init__(message)
+        logging.error(message)
```

Comment:
```suggestion
        logging.error(f" {message}")
```

Doing this so that there is a space after `:` as it is not done by the method itself.